lib/topics.ts

```diff
@@ -0,0 +1,16 @@
+import statusGreen from '../public/images/status-green.svg';
+import statusAmber from '../public/images/status-amber.svg';
+import statusRed from '../public/images/status-red.svg';
+
+import type { StatusColorChoice } from '../components/StreamField/blocks/TopicsBlock/types';
+
+const TopicStatusImages = {
+    green: statusGreen,
+    red: statusRed,
+    amber: statusAmber,
+};
+
+const getTopicStatusImage = (statusColor: StatusColorChoice) =>
+    TopicStatusImages[statusColor];
```

Comment:
**Suggestion**

This is just my preference but I don't think this function is necessary. We can just export `topicStatusImages` and immediately get the value from there instead of through a function. No worries if you'd rather keep this in.